### PR TITLE
Add YoloV6 R1 Backward Support

### PR DESCRIPTION
**Pull request by HonzaCuhel**
Added support for YoloV6 R1 as separate service.
Screenshot from 2023-04-26 01:10:15

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -86,7 +86,8 @@ function App() {
                         <select id="version" value={config.version} name="version" className="form-select" aria-label="Default select example"
                                 onChange={e => update({version: e.target.value})}>
                           <option value="v5">YoloV5</option>
-                          <option value="v6">YoloV6</option>
+                          <option value="v6">YoloV6 R1</option>
+                          <option value="v6r2">YoloV6 R2 & R3</option>
```
Comment (tersekmatija) [inline client/src/App.js:90]: Let's perhaps change this to:
YoloV6 (latest) and YoloV6 (legacy - R1).
And let's move legacy one to the bottom.

File: client/src/store.js [hunk 2]
```diff
@@ -57,11 +65,13 @@ export const upload = createAsyncThunk(
...
       }
       switch (status) {
+        case 517:
+          throw Error("Error while loading model (This may be caused by trying to convert a newer version of YoloV6 - release 2.0 or 3.0, if that is the case, try to convert using the `YoloV6 R2 & R3` option).");
         case 518:
           let errorData = JSON.parse(String.fromCharCode.apply(String, new Uint8Array(error.response.data)))
           throw Error(errorData['message']);
         case 519:
-          throw Error("Error while loading model (This may be caused by trying to convert an older version of YoloV6 - release 1.0, if that is the case, we are working on a fix at the moment)");
+          throw Error("Error while loading model (This may be caused by trying to convert an older version of YoloV6 - release 1.0, if that is the case, try to convert using the `YoloV6 R1` option).");
```
Comment (tersekmatija) [inline client/src/store.js:74]: Update as mentioned above.

File: client/src/store.js [hunk 3]
```diff
@@ -57,11 +65,13 @@ export const upload = createAsyncThunk(
         console.log(JSON.stringify(error, null, 4));
       }
       switch (status) {
+        case 517:
+          throw Error("Error while loading model (This may be caused by trying to convert a newer version of YoloV6 - release 2.0 or 3.0, if that is the case, try to convert using the `YoloV6 R2 & R3` option).");
```
Comment (tersekmatija) [inline client/src/store.js:69]: Update names as mentioned above.

**Review by tersekmatija — APPROVED** [review]
Thanks, looks good. Were all V6 versions tested?

File: client/src/App.js (hunk 1 above)
Comment (HonzaCuhel) [reply client/src/App.js:90]: So that YoloV6 (legacy - R1) would be the lowest option?

**Comment by HonzaCuhel** [comment]
Yes, I tested all R1 versions - nano, tiny, small, and then nano versions of R2, R2.1, and R3 (and also V6m R3)

File: client/src/App.js (hunk 1 above)
Comment (tersekmatija) [reply client/src/App.js:90]: Yes @HonzaCuhel

**Comment by HonzaCuhel** [comment]
Updated

Screenshot from 2023-04-28 14:49:47

**Pull request merged**